tools/ci_build/github/windows/bundle_dlls_gpu.bat

```diff
@@ -10,6 +10,11 @@ FOR /R %%i IN (*.nupkg) do (
        mkdir runtimes\linux-x64\native
        move onnxruntime-linux-x64\lib\libonnxruntime.so.1* runtimes\linux-x64\native\libonnxruntime.so
        move onnxruntime-linux-x64\lib\libonnxruntime_providers_* runtimes\linux-x64\native
+       mkdir runtimes\win-x64\native
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime_providers_tensorrt.dll runtimes\win-x64\native\onnxruntime_providers_tensorrt.dll
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime.dll runtimes\win-x64\native\onnxruntime.dll
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime.lib runtimes\win-x64\native\onnxruntime.lib
+       move onnxruntime-win-tensorrt-x64\lib\onnxruntime.pdb runtimes\win-x64\native\onnxruntime.pdb
        7z a  %%~ni.nupkg runtimes
    )
-)
\ No newline at end of file
+) 
```

Comment:
Linux TRT files are not there!

Reply:
Do we have linux pool that has TRT8 library with it now?

Reply:
Added.